### DISCUSSION TITLE
Feature Request to use "to" instead of "href" in "<Link></Link>" from "next/link"

### DISCUSSION THREAD

**Discussion by test4appdev1** [vercel/next.js#34220] [Ideas]
For **`"<Link></Link>"` from "react-router-dom", ReactJS**,  **"to"** is used as shown below:

```js
import { Link } from 'react-router-dom';

const Hello = () => {
  return (
    <Link to='/hello'>Hello</Link> // "to" is used
  );     
};

export default Hello;
```

But for For **`"<Link></Link>"` from "next/link", NextJS**,  **"href"** is used as shown below:

```js
import Link from 'next/link';

const Hello = () => {
  return (  
    <Link href='/hello'> // "href" is used
      Hello
    </Link>    
  )
}

export default Hello;
```

It's really confusing when I switch between **ReactJS** and **NextJS**.

**Comment by merelinguist** [comment]
React Router: `to="/"`
Next.js: `href="/"`
HTML: `href="/"`

Why should Next change their API just to imitate an unrelated library?
